tests/tensorflow/test_tensorflow2_autolog.py

```diff
@@ -324,6 +324,89 @@ def test_tf_keras_autolog_implicit_batch_size_works(generate_data, batch_size):
     assert mlflow.last_active_run().data.params["batch_size"] == str(batch_size)
 
 
+def __example_tf_dataset_multi_input(batch_size):
```

Comment:
```suggestion
def __tf_dataset_multi_input(batch_size):
```

Let's remove `example`.

Reply:
changed 👍